### DISCUSSION TITLE
Chrome的无障碍即时字幕翻译

### DISCUSSION THREAD

**Discussion by ggyy020406** [immersive-translate/immersive-translate#1554] [Ideas]
### 新功能 | New Feature

1.Chrome的无障碍即时字幕双字幕
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.学习外语
2.
3.

### 提议 | Proposal

现在Chrome的无障碍即时字幕已经自带翻译功能了，可以令它双字幕吗？

**Comment by theowenyoung** [comment]
插件没有权限读取。
